lib/tasks/db.rake

```diff
@@ -0,0 +1,49 @@
+namespace :db do
+
+  desc 'Dumps the database to file'
+  task :dump => :environment do
+
+    host = ActiveRecord::Base.connection_config[:host]
+    db = ActiveRecord::Base.connection_config[:database]
+
+    cmd = "pg_dump --no-owner --no-acl -v -f #{dump_file} --no-password "
```

Comment:
Also, if we add `-c` as a parameter to `pg_dump`, we won't have to do a destroy beforehand as it will drop before creating anyway.